docs/orchestrating-elastic-stack-applications/accessing-elastic-services.asciidoc

```diff
@@ -148,10 +149,22 @@ Create a Kubernetes secret with:
 
 [source,sh]
 ----
-kubectl create secret generic my-cert --from-file=ca.crt=tls.crt --from-file=tls.crt=tls.crt --from-file=tls.key=tls.key
+kubectl create secret generic my-cert --from-file=ca.crt --from-file=tls.crt --from-file=tls.key
 ----
 
-Then, you have to reference the secret name in the `http.tls.certificate` section of the resource manifest.
+Alternatively you can also bring your own CA certificate including a private key and let ECK issue certificates with it.
```

Comment:
nit: I think I would add a comment to explain how `selfSignedCertificate` is used in the generated certificates.

Reply:
This is actually one of the not so great aspects of this change. We will have a an attribute `selfSignedCertificate.subjectAltNames` that affects also certificates that are not actually self-signed.